cmd/main.go

```diff
@@ -123,6 +123,9 @@ func buildCmdTree() *cobra.Command {
 	// kubebuilder version
 	rootCmd.AddCommand(version.NewVersionCmd())
 
+	// kubebuilder completion
+	rootCmd.AddCommand(newCompletionCmd())
```

Comment:
why it was added here?

Reply:
This is the `kubebuilder` command root, so this line creates the invocation string:
```sh
$ kubebuilder completion <bash|zsh>
```

Reply:
Yes, make sense. Tks. I saw now .

Reply:
Solved you can close.